rulebook.tex

```diff
@@ -187,49 +216,48 @@
+widely accepting a new paradigm for organizing production by introducing
+perceiving, active, context-aware autonomous systems. This paradigm is
+often referred to as Industry~4.0~\cite{Industry-4.0}, a move from static
+process chains towards more automation and autonomy. One of its corner stones
+are so called \emph{smart factories} which are flexible facilities in which
+manufacturing steps are considered as services. This means that various
+production steps, machines and sub-facilities can be combined in (almost)
+arbitrary ways allowing for the production of various product types and
+variants cost-effectively, even in small quantities. This is in contrast To
+traditional production chains which produce only a small number of varieties
+of a product at high quantities~\cite{RCLL-CPS}. To realize such factories
+and make use of their full potential, more capable logistics systems are needed,
+of which the most flexible form are autonomous mobile robots.
+
+The \ac{RCLL} is determined to develop into a
 state-of-the-art platform for mobile robotics education and research
-tackling this problem of flexible and efficient production logistics at a
-comprehensible size. This industrial motivated league focuses
+tackling the problem of flexible and efficient production logistics at a
+comprehensible size. Being an industry motivated league, it focuses
 on challenges promoting precise actions and robust long-enduring
 execution, and further encourages external data supported
-autonomy. By providing feedback and competition hardware, Industrial
-partner Festo provides insights into future factory concepts to help
+autonomy. By providing feedback and competition hardware, industrial
+partner Festo transfers insights into future factory concepts to help
  uncover relevant future topics for education and research.
 
  This year's competition is laid out in the following pages.  It ensures
- the same and fair circumstances for all participants. It neither
- dictates nor suggests the way how to fulfill the task, but is meant
- to develop the RCLL further. Smart Factory environments require new
+ the same and fair conditions for all participants. It neither
+ dictates nor suggests how to fulfill the task, but is meant
+ to encourage new developments in the \ac{RCLL} through restrictions
+ and tasks. Smart Factory environments require new
  concepts of flexibility, awareness and optimization from autonomous
  entities working in an environment with specified, yet to some degree
  uncertain and dynamic, agency. This includes current challenges of
  developing industry-wide standards for Cyber Physical Systems for
  production processes like designing plug-and-produce capable
- systems. Opposed to numerous approaches regarding production
- infrastructure, RCLL aims to provide concepts for smart production
- logistics.
+ systems.
 
  After exciting competitions in the previous years, we look forward to a new
```

Comment:
Maybe making this its own subchapter, called "History"?

Reply:
Good idea, done int https://github.com/robocup-logistics/rcll-rulebook/pull/43/commits/ff88dcc1e0e281617ab7af3410888e3f935e3eed